.rubocop_todo.yml

```diff
@@ -7,13 +7,13 @@ require:
 
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2020-04-02 16:59:26 +0200 using RuboCop version 0.79.0.
+# on 2020-04-10 07:46:58 +0200 using RuboCop version 0.79.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-# Offense count: 252
+# Offense count: 258
```

Comment:
This should be going down, not up 😢 Thanks for the effort you put into this @rhymes 💯

Reply:
hahaha yeah :D One day we'll start addressing this file, hopefully